__tests__/app.test.js

```diff
@@ -287,16 +287,49 @@ describe('/api/articles/:article_id/comments', () => {
         })
     })
 
-    test('POST:400 responds with an 400 and msg when provided with an incomplete object (no body)', () => {
+    test('POST 400: responds with an 400 and msg when provided with an incomplete object (no body)', () => {
         return request(app)
         .post('/api/articles/3/comments')
         .send({
             username: "butter_bridge"
         })
         .expect(400)
         .then(({body}) => {
-            expect(body.msg).toBe('Bad request');
-        });
-      });
+            expect(body.msg).toBe('Bad request')
+        })
+      })
+})
+
+describe('/api/comments/comment_id', () => {
+    test('DELETE 200: Responds with status 204 and no content, removes comment of given comment_id', () => {
+        return request(app)
+        .delete('/api/comments/1')
+        .expect(200)
```

Comment:
We can be more specific with the error code here, generally you would see a 204 no content as a response from a delete